google/cloud/experimental_tag.h

```diff
@@ -0,0 +1,30 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef GOOGLE_CLOUD_CPP_GOOGLE_CLOUD_EXPERIMENTAL_TAG_H
+#define GOOGLE_CLOUD_CPP_GOOGLE_CLOUD_EXPERIMENTAL_TAG_H
+
+#include "google/cloud/version.h"
+
+namespace google {
+namespace cloud {
+GOOGLE_CLOUD_CPP_INLINE_NAMESPACE_BEGIN
+
+struct ExperimentalTag {};
```

Comment:
can we say what this thing means?

"If this tag is present as a function parameter, the function is experimental. Experimental functions are subject to change without notice, including removal."

Reply:
s/, they/. They/